include/common/mir/events/event_builders.h

```diff
@@ -95,6 +99,20 @@ void add_touch(
     float touch_minor_value,
     float size_value);
 
+// Pointer event with all properties, using Mir geometry types
+EventUPtr make_pointer_event(
+    MirInputDeviceId device_id,
+    std::chrono::nanoseconds timestamp,
+    std::vector<uint8_t> const& cookie,
+    MirInputEventModifiers mods,
+    MirPointerAction action,
+    MirPointerButtons buttons,
+    std::optional<mir::geometry::PointF> position,
+    mir::geometry::DisplacementF motion,
+    MirPointerAxisSource axis_source,
+    events::ScrollAxisV1<mir::geometry::DeltaXTag> h_scroll,
```

Comment:
Some places you use `ScrollAxisV1<Tag>`, some places you use `ScrollAxis<Tag>`, and some places you use `ScrollAxisH`/`ScrollAxisV`. There's presumably some reason behind this, but I can't see it?

Reply:
How it should be is `ScrollAxisV1<Tag>` is used in places where we don't want the version auto-updated (for example, in function signatures of ABI-stable functions), `ScrollAxis<Tag>` is used in places that handle the tag generically but should always use the latest version and `ScrollAxisH`/`ScrollAxisV` are used most places where neither of those are the case. As it is now there are residual uses of `ScrollAxis<Tag>`, which I'll fix.